udata/core/organization/api.py

```diff
@@ -87,8 +87,13 @@ def get(self, org):
     @api.expect(org_fields)
     @api.marshal_with(org_fields)
     @api.response(400, errors.VALIDATION_ERROR)
+    @api.response(410, 'Organization has been deleted')
     def put(self, org):
-        '''Update a organization given its identifier'''
+        '''
+        Update a organization given its identifier
+
+        :raises PermissionDenied:
+        '''
         if org.deleted:
             api.abort(410, 'Organization has been deleted')
```

Comment:
should https://github.com/etalab/udata/pull/451/commits/45501bf92e4542163142d50664826b5daeec1941#diff-6ce9a6ba822a15f73a6b9269b1a7c62aR109 also get a decorator?